frontend/src/lib/components/IntervalFilter/IntervalFilter.tsx

```diff
@@ -6,15 +6,19 @@ import { disableHourFor, disableMinuteFor } from 'lib/utils'
 import { CalendarOutlined } from '@ant-design/icons'
 import { defaultInterval, IntervalKeyType, intervals } from 'lib/components/IntervalFilter/intervals'
 import { InsightType } from '~/types'
+import { insightLogic } from 'scenes/insights/insightLogic'
 
 interface InvertalFilterProps {
     view: InsightType
     disabled?: boolean
 }
 
 export function IntervalFilter({ view, disabled }: InvertalFilterProps): JSX.Element {
-    const { interval } = useValues(intervalFilterLogic)
-    const { setIntervalFilter, setDateFrom } = useActions(intervalFilterLogic)
+    const { insightProps } = useValues(insightLogic)
+    const {
+        filters: { interval },
+    } = useValues(intervalFilterLogic(insightProps))
+    const { setIntervalFilter, setDateFrom } = useActions(intervalFilterLogic(insightProps))
```

Comment:
This `setDateFrom` got recently removed in https://github.com/PostHog/posthog/pull/7758 ... when all that code got moved into a listener.